packages/@aws-cdk/aws-stepfunctions/test/integ.custom-state.ts

```diff
@@ -0,0 +1,35 @@
+import * as cdk from '@aws-cdk/core';
+import * as sfn from '../lib';
+
```

Comment:
I usually like to provide a comment on how to verify that the integ test worked - https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests. 

In this case, if you can print the state machine ARN as a `CfnOutput`, then it's calling start-execution and checking with describe-execution. See the [example here](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-stepfunctions-tasks/test/batch/integ.run-batch-job.ts#L9-L16)

Reply:
@nija-at I haven't forgotten about that - In this case, we're depending on a table that we aren't creating so although this will deploy successfully, it won't actually run succesfully without creating the table. Perhaps the validation steps should create the table and then it'll work? 

The thorough example is here in this [gist](https://gist.github.com/shivlaks/2dea2bcc318d5d4067258a6f114fbfb1), but I'd prefer if this integ test was in the aws-stepfunctions-tasks package since those deps are readily available there.

Alternatively, Is there a simpler way to bring in dependencies (such as dynamodb) for testing?

Reply:
adding the example with the stateARN, I'm going to see if we can do it in a custom resource (outside this PR)